### PR TITLE
Decrease min channel name length to 3

### PR DESCRIPTION
**Pull request by hpeebles**

### CODE REVIEW

File: frontend/app/src/components/home/communities/edit/ChooseChannels.svelte [hunk 1]
```diff
@@ -8,7 +8,7 @@
     import EditableChannel from "./EditableChannel.svelte";
     import ErrorMessage from "../../../ErrorMessage.svelte";
 
-    const MIN_CHANNEL_LENGTH = 4;
```
Comment (megrogan) [inline frontend/app/src/components/home/communities/edit/ChooseChannels.svelte:11]: The backend change also includes group and community names

Comment (hpeebles) [reply frontend/app/src/components/home/communities/edit/ChooseChannels.svelte:11]: For groups it is already 3

Comment (megrogan) [reply frontend/app/src/components/home/communities/edit/ChooseChannels.svelte:11]: Community names have a min of 4 on FE

Comment (megrogan) [reply frontend/app/src/components/home/communities/edit/ChooseChannels.svelte:11]: I can sort this

**Pull request closed without merging**